### DISCUSSION TITLE
Pinch to zoom

### DISCUSSION THREAD

**Discussion by acmilan77051** [sumatrapdfreader/sumatrapdf#3758] [General]
On my Surface Pro 7+, if I try to pinch to zoom inside the app, it either scrolls down or up. Doesn't zoom. It's bothering me very much. It's such a lightweight app that, does what i need, except this issue. What am i missing here? I could not find anything in the settings. Please help me out.

**Comment by kjk** [comment]
Which version are you using? What exactly do you mean by "pinch to zoom"?

I just tested latest [pre-release](https://www.sumatrapdfreader.org/prerelease) using pinch zoom on touchscreen on my ThinkPad x1 and it zooms as expected.

**Comment by acmilan77051** [comment]
on touchscreen is good, but i can't do pinch to zoom on touchpad of my surface pro 7.

**Reply by GitHubRulesOK** [reply]
Hmm my very old windows 7 touchscreen & touchpad  will zoom but there are system settings that need to be set or adjusted so its very much a hardware feature that may need looking at. Pinch on pad is very fickle and crude so I use the screen for fine smooth scroll & zoom its a bigger touch pad :-)
